source/uwp/Renderer/lib/AdaptiveFact.cpp

```diff
@@ -21,6 +21,8 @@ AdaptiveNamespaceStart
     HRESULT AdaptiveFact::RuntimeClassInitialize(const std::shared_ptr<AdaptiveSharedNamespace::Fact>& sharedFact)
     {
         m_sharedFact = sharedFact;
+        RETURN_IF_FAILED(UTF8ToHString(sharedFact->GetValue(), m_value.GetAddressOf()));
+        RETURN_IF_FAILED(UTF8ToHString(sharedFact->GetLanguage(), m_language.GetAddressOf()));
```

Comment:
Let's be consistent here. Either reference all properties directly out of m_sharedFact, or else make all properties separate member variables. We dont' want to have value and language as separate members while title is stored in the shared fact.